lib/KImg/__tests__/KImg.spec.js

```diff
@@ -23,26 +20,20 @@ describe('KImg', () => {
   });
 
   it(`throws an error when no 'altText' is provided`, () => {
-    const error = jest.fn();
-    makeWrapper({
-      propsData: { src: '/le-logo.svg', altText: undefined },
-      listeners: { error },
-    });
-    expect(error).toHaveBeenCalled();
-    expect(error.mock.calls[0][0]).toBeInstanceOf(Error);
-    expect(error.mock.calls[0][0].message).toBe(
-      'Missing required prop - provide altText or indicate isDecorative'
-    );
+    expect(() =>
```

Comment:
Hi @KshitijThareja! I think we can let this file without modifications. In this case it seems it is the expected behaviour to pass an error listener to catch these errors. See https://github.com/learningequality/kolibri-design-system/pull/645#discussion_r1617393477.

Reply:
Sure. I'll make the changes. I was currently working on finding a way to use `concurrently` as discussed, so was a bit busy with that. I'll get this resolved by today at the earliest.